packages/beat/manifest.yml

```
@@ -0,0 +1,32 @@
+name: beat
+title: Beat
+version: 0.0.1
+description: Beat Integration
+type: integration
+conditions:
+  kibana.version: ^8.7.0
```

You can add a condition here if you would like to keep the equivalent to `license: basic` in the format version 1.0.0.
```suggestion
  kibana.version: ^8.7.0
  elastic.subscription: basic
```